blocked-edges/4.12.54-RHELKernelCephFSCapDropPanic.yaml

```diff
@@ -0,0 +1,15 @@
+to: 4.12.54
+# 4.12 before 4.12.54 and 4.11
+from: 4[.](11[.].*|12[.]([1-4]?[0-9]|5[0123]))[+].*
+url: https://issues.redhat.com/browse/COS-2705
+name: RHELKernelCephFSCapDropPanic
+message: "Nodes in clusters running workloads that mount Ceph volumes may experience kernel panics due to a CephFS client bug"
+matchingRules:
+- type: PromQL
+  promql:
+    promql: |
+      topk(1,
+        label_replace(group(ceph_health_status), "ceph", "yes", "", "")
+        or
+        label_replace(0 * group(cluster_version), "ceph", "no", "", "")
```

Comment:
nit: can you add #3591's `_id=""` for HyperShift-compatibility to `cluster_version` and `ceph_health_status`?